docs/book/src/topics/cloud-provider-config.md

```diff
@@ -5,3 +5,31 @@ The [Azure cloud provider](https://github.com/kubernetes-sigs/cloud-provider-azu
 CAPZ automatically generates this file based on user-provided values in AzureMachineTemplate and AzureMachine. All AzureMachines in the same MachineDeployment or control plane will all share a single cloud provider secret, while AzureMachines created inidividually will have their own secret.
 
 For AzureMachineTemplate and standalone AzureMachines, the generated secret will have the name "${RESOURCE}-azure-json", where "${RESOURCE}" is the name of either the AzureMachineTemplate or AzureMachine. The secret will have one data field, `azure.json`, with the raw content for that file. When the secret `${RESOURCE}-azure-json` already exists in the same namespace as an AzureCluster and does not have the label `"${CLUSTER_NAME}": "owned"`, CAPZ will not generate the default described above. Instead it will directly use whatever the user provides in that secret.
+
+# External Cloud Provider
+
+To deploy a cluster using [external cloud provider](https://github.com/kubernetes-sigs/cloud-provider-azure), create a cluster configuration with the [external cloud provider template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/templates/cluster-template-external-cloud-provider.yaml).
+
+After components are deployed, you should see following pods in `Running` state:
```

Comment:
```suggestion
After deploying the cluster, you should eventually see a set of pods like the following in a `Running` state:
```

Reply:
Not sure if we wanted any actual content changes here, so feel free to ignore this--I can submit a separate PR.

Reply:
applied as a separate commit